pkg/agent/openflow/multicast.go

```diff
@@ -81,7 +82,7 @@ func (f *featureMulticast) multicastReceiversGroup(groupID binding.GroupIDType,
 		group = group.Bucket().
 			LoadToRegField(OFPortFoundRegMark.GetField(), OFPortFoundRegMark.GetValue()).
 			LoadToRegField(TargetOFPortField, ports[i]).
-			ResubmitToTable(tableID).
+			ResubmitToTable(MulticastIngressPodMetricTable.GetID()).
```

Comment:
You can't replace the tableID inside the function derectly, otherwise, both IGMP Query group and multicast forwarding flows are impacted. Please modify the table ID in the caller function. I guess you want to modify `InstallMulticastGroup`?

Reply:
updated